### PR TITLE
Add Hugo as static website engine for Enclave

### PR DESCRIPTION
**Pull request by blankoworld**

### CODE REVIEW

File: website/themes/ecoloJ/layouts/partials/head.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+  <title>{{ .Page.Title }}</title>
+	<style>
+		@media print {
+			#ghostery-purple-box {
```
Comment (Lukc) [inline website/themes/ecoloJ/layouts/partials/head.html]: Is this code related to a browser add-on?

Comment (blankoworld) [reply website/themes/ecoloJ/layouts/partials/head.html]: Oops, probably. I will fix it soon.

**Comment by blankoworld** [comment]
Should be OK now
